src/interaction/InteractionManager.js

```diff
@@ -414,12 +436,47 @@ InteractionManager.prototype.processMouseDown = function ( displayObject, hit )
 
     if(hit)
     {
-        displayObject[ isRightButton ? '_isRightDown' : '_isLeftDown' ] = true;
-        this.dispatchEvent( displayObject, isRightButton ? 'rightdown' : 'mousedown', this.eventData );
+        // only when the double click callback is defined we should perform
+        // the logic for checking double clicks
+        if(displayObject.dblclick)
```

Comment:
This won't work if I do `displayObject.on('dblclick', ...);` since that doesn't set the event property, just adds a listener.